app/code/Magento/InventoryApi/Test/_files/source_items_out_stock.php

```diff
@@ -0,0 +1,69 @@
+<?php
```

Comment:
Need to merge develop brunch and adapt tests according to possibility to pre-define id

Reply:
Need to simplify fixture files
https://github.com/magento-engcom/magento2/pull/76/files#diff-c8b20cde2aec460219d830973d4b6321
like in
https://github.com/magento-engcom/magento2/blob/develop/app/code/Magento/InventoryApi/Test/_files/source_items.php

Now it is possible after merge
https://github.com/magento-engcom/magento2/pull/79
We try to describe more details in our article
https://github.com/magento-engcom/magento2/wiki/The-first-step-towards-pre-generated-IDs.-And-how-this-will-improve-your-Integration-tests